examples/juice-shop.codetf-v3.json

```diff
@@ -0,0 +1,181 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://pixee.ai/codetf-v3.schema.json",
+  "title": "Example Code Transformation Format (CodeTF) v3.0",
+  "run": {
+    "vendor": "pixee",
+    "tool": "codemodder-python",
+    "version": "0.1.0",
+    "elapsed": 229992,
+    "commandLine": "codemodder-python --verbose --dry-run --sonar /Users/example/sonar_juice-shop.json /Users/example/juice-shop",
+    "directory": "/Users/example/juice-shop",
+    "sarifs": []
+  },
```

Comment:
I would expect this to be a single `fix`. I thought we liked the idea of fix orchestration being done upstream. This implies there should be some orchestration here.

This means callers would have to change how they call codemods -- with a single input, but that significantly de-scopes the codemodder implementations.